### DISCUSSION TITLE
Excersise Time

### DISCUSSION THREAD

**Discussion by bcrazycramer** [jgyates/genmon#1100] [General]
Hello,

Is it possible to extend the excersise time? 

We are running Generac G0070390.

Thanks,
Brandon

**Comment by jgyates** [comment]
Not for that model. For Generac Evolution the only controllers that support a variable exercise time are the liquid cooled models and then it is only a range of 12 to 60 seconds. You can simulate adding a second exercise cycle with the Enhanced Exercise add on.

**Comment by skipfire** [comment]
What is your goal?  Enhanced exercise add-on may get what you want, though not what you specifically asked for.

**Comment by bcrazycramer** [comment]
It looks like the Add On enhanced excerise will run the generator for whatever time/duration I set it for. It just ran for a 1/2 hour. Does this replace the normal built in excersise time? Do I just turn that off?

**Reply by jgyates** [reply]
No, it adds a 2nd exercise cycle. There is no way to disable the built in exercise cycle.

**Reply by skipfire** [reply]
I use mine in addition to the standard exercise and use mine for load testing (the transfer setting).  I have my regular exercise every Monday at 10 AM (so 10:00 to 10:05) and then my enhanced exercise on the first of each month 9:30-10:30 so on April first when the two matched up, I only had the transfer test and not the weekly test, so if you schedule them to overlap and make sure the enhanced one starts before and ends after the built-in, you would effectively eliminate the regular one.

**Comment by Jkh62** [comment]
I will add prior to getting Genmon installed and operating my 22KW Generac had VERY milky oil , lot of condensation. I did the 1st oil and filter change and watched it and it was not going away doing the Generac default exercise run time ( at full RPM ).

Running Genmon I manually run a 30 minute exercise time under load  and no trace of milky oil anywhere.
This was the biggest motivating factor for me to get Genmon running.

Not sure if a want to use the extended run as I like to watch the voltages and other output monitors but manual or automatic with extended run by schedule your engine will thank you.

**Reply by biggen1684** [reply]
How often are you running that 30 minute loaded exercise?

**Comment by Jkh62** [comment]
The 1st of every month.
Since its been in the 90's I ran it for 20 minutes but my engine oil is clean and not a trace of water / condensation since running it under load for 30 minutes , winter or summer.

Just got a chance to see what my central AC unit pulls under load off gen power. Carrier  3 1/2 ton HVAC.

Start ~

https://i.imgur.com/1YJPmSS.jpg

Running~

https://i.imgur.com/a7SoPaQ.jpg
